lib/css/components/post-summary.less

```diff
@@ -1,436 +1,407 @@
-        flex-direction: row;
-        align-items: center;
-        width: auto;
-    });
-}
+    // Child Elements
+    & & {
+        &--answer {
+            & + & {
+                margin-top: var(--su16);
+            }
 
-.s-post-summary--content {
-    flex-grow: 1;
-    max-width: 100%;
-
-    .s-post-summary--content-title {
-        display: block;
-        font-size: var(--fs-body3);
-        margin-top: -0.15rem; // Optical alignment to compensate for title's containing block
-        margin-bottom: 0.3846rem;
-        padding-right: var(--su24);
-        line-height: var(--lh-md);
-        font-weight: normal;
-        .break-word;
-
-        .iconShield {
-            color: var(--fc-light);
-        }
+            &:before {
+                .highcontrast-mode({
+                    background: var(--black-600);
+                });
+
+                background: var(--black-150);
+                border-radius: var(--su8);
+                bottom: 0;
+                content: "";
+                display: block;
+                left: 0;
+                position: absolute;
+                top: 0;
+                width: var(--su4);
+            }
 
-        .svg-icon {
-            vertical-align: text-bottom; // Optical alignment
+            margin: var(--su16) 1em 0 1em;
+            padding: 0.5em 0 0.5em calc(1em + var(--su4));
             position: relative;
-            top: -1px;
         }
 
-        a {
-            .break-word;
-
-            color: var(--theme-post-title-color);
-            font-family: var(--theme-post-title-font-family);
-
-            &:visited {
-                color: var(--theme-post-title-color-visited);
-            }
+        &--answer-excerpt {
+            .v-truncate4;
+            color: var(--black-600);
+            margin-bottom: var(--su8);
+        }
 
-            &:hover,
-            &:active {
-                color: var(--theme-post-title-color-hover);
-            }
+        &--content {
+            flex-grow: 1;
+            max-width: 100%;
         }
-    }
 
-    .s-post-summary--content-type {
-        color: var(--fc-medium);
-        margin-bottom: var(--su4);
+        &--content-excerpt {
+            &.s-post-summary--content-excerpt {
+                &__sm {
+                    .v-truncate1;
+                }
 
-        .svg-icon {
-            margin-left: calc(var(--su2) * -1);
-            color: var(--fc-light);
-        }
-    }
+                &__md {
+                    .v-truncate3;
+                }
 
-    .s-post-summary--content-excerpt {
-        margin-top: calc(var(--su2) * -1);
-        margin-bottom: var(--su8);
-        font-family: var(--theme-post-body-font-family);
-        color: var(--fc-medium);
-        .v-truncate2;
-        .break-word;
+                &__lg {
+                    .v-truncate4;
+                }
+            }
 
-        &.s-post-summary--content-excerpt__sm {
-            .v-truncate1;
+            .break-word;
```

Comment:
Just thinking out loud: in this file we make extensive usage of "atomic classes as mixins". I don't have strong opinions about that but I was wondering if we could do something similar also when we will write the styles of our new WCs; it is certainly handy to extend/refer the rules like that instead of rewriting them over and over, at the same time though we create "implicit" coupling across 2 less files. I think it would be more maintainable if there would be an "explicit" import at the beginning of the file showing that this file depends on typography.less (something like `@use typography.less as t` in SCSS I believe). No action required here just food for thoughts in terms of maintainability. 🙂

Reply:
I like the explicit import idea. Seeing as we're considering splitting up components/atomic classes/etc into standalone repos, it seems like a natural progression.